kubetest/kind/kind.go

```diff
@@ -333,10 +338,6 @@ func (d *Deployer) DumpClusterLogs(localPath, gcsPath string) error {
 
 // TestSetup is a NO-OP in this deployer.
 func (d *Deployer) TestSetup() error {
-	// set KUBECONFIG
-	if err := d.setKubeConfigEnv(); err != nil {
```

Comment:
hmmm what if the user did `kubetest --deployment=kind --up && kubetest --deployment=kind --test` ?

Reply:
`_ := d.setKubeConfigEnv()`
in `TestSetup()`?

Reply:
yeah I think that's the solution